metadata-ws/src/test/resources/features/study.feature

```diff
@@ -1,22 +1,36 @@
 Feature: study object
 
-  Scenario: register a study successfully
-    When I request POST /taxonomies with JSON payload:
+  Scenario: register a study successfully and check its fields
+    # Create a taxonomy
+    When I request POST taxonomies with 9606 for ID, Homo Sapiens for name and TAXONOMY_1 for ancestors
```

Comment:
I wonder where is TAXONOMY_1 is created?

Reply:
I guess there is no validation for ancestors so it passed, so you can make it NONE or create TAXONOMY_1 and refer to it

Reply:
You're right! Fixed the ancestor taxonomy to `NONE`

Reply:
Why doesn't the application code fail when trying to use a non-existing ancestor taxonomy?

Reply:
Because this is the way it's currently implemented:

```
    @ManyToMany
    @JsonProperty
    private List<Taxonomy> ancestors;
```

There's no `@NotNull` constraint, hence the `ancestor` list is allowed to contain `null` values

Reply:
And we can't just trivially fix this because one "root" taxonomy entity must still be present

Reply:
Right, I was thinking more on the lines of pointing to a taxonomy that does not exist. But I guess the model should control that.